### DISCUSSION TITLE
同时存在两个版本的zotero

### DISCUSSION THREAD

**Discussion by eeendless** [zotero-chinese/wiki#327] [求助 / Q&A]
![屏幕截图 2024-12-13 211017](https://github.com/user-attachments/assets/bf9b75ab-bb70-4ce0-ba7b-78ce517f238d)
如图同时存在两个版本的zotero，尝试卸载低版本但会将两个版本同时卸载。之前也出现过，甚至一个版本是32位，一个64位，全部卸载重装后又出现这个问题。请问是bug吗？有什么解决方法？🙏

**Comment by vincentlee10001** [comment]
我也是这个问题。用Hbituninstaller查了一下注册表，好像是这两个版本的注册表同时存在，我没敢删怕出问题

**Reply by eeendless** [reply]
我发现我一个是安装在c盘，一个安装在d盘，但是注册表好像是一个。我不记得在哪看到，安装是要以管理员模式运行，我就这样直接覆盖安装到c盘，现在只有一个版本了。不过我没有试以管理员模式安装到d盘，想着zotero也没多大就这样了，您可以尝试一下

**Reply by Dawsine** [reply]
> 我发现我一个是安装在c盘，一个安装在d盘，但是注册表好像是一个。我不记得在哪看到，安装是要以管理员模式运行，我就这样直接覆盖安装到c盘，现在只有一个版本了。不过我没有试以管理员模式安装到d盘，想着zotero也没多大就这样了，您可以尝试一下

我刚刚尝试了，这个方法是有效的，只要使用管理员模式重新安装，你甚至可以安装到新的位置，然后旧版本就会完全失效

**Comment by xianchengyue** [comment]
<img width="1044" height="213" alt="image" src="https://github.com/user-attachments/assets/87527354-cb17-436f-8b62-6280e809bd3d" />
我有三个版本，而且都是用一个安装位置，也不敢卸

**Comment by windbd** [comment]
我也是，同一个安装位置，不同的版本

**Comment by snailwd** [comment]
我也有这个问题，想着卸载一个，于是现在重装中……

**Comment by silentmomo03** [comment]
我也有两个版本的zotero，也是路径相同，不知道咋办，不办难受

**Comment by Alien-2022** [comment]
我也是同样的问题，同时存在7.x和8.x的版本，解决办法是先在官网下载一个最新的安装包以管理员身份运行，会提示电脑存在32位的版本如果继续会移除，点继续，然后选择当前自己在用的zotero版本（比如我选8.x）安装目录安装就好了，数据也不会丢失。

**Reply by ElishaQuan** [reply]
解决了！谢谢！亲测这个办法完美解决。

**Reply by Alien-2022** [reply]
好耶
> 解决了！谢谢！亲测这个办法完美解决。
>

**Reply by wwhh999** [reply]
有用！

**Reply by likwater** [reply]
有用

**Comment by p1aintiff** [comment]
为什么还没修复（大哭）

**Comment by DejaVuDarren** [comment]
删除那个旧版本的注册表就可以了，不会的可以问AI怎么删。不会影响新版本的软件。
